root_orchestrator/resource-abstractor/db/clusters_db.py

```diff
@@ -1,7 +1,24 @@
+from datetime import datetime
+
 import db.mongodb_client as db
 from bson.objectid import ObjectId
+from db.clusters_helper import get_freshness_threshhold
```

Comment:
Nitpick: typo
-> threshold

Reply:
Thanks! I actually always thought its `threshhold` similar to `withhold`. English is stupid 😄

Reply:
Oh yes it is 😅